Tasks/ChocoTask/task.yaml

```diff
@@ -1,3 +1,15 @@
+# This task installs choco package files
+#
+# USAGE
+#
+# - task: choco
+#     inputs:
+#       packages: "vscode, azure-cli"
+#       Versions: "{ 'vscode'='1.80.0'; 'azure-cli'='2.50.0' }"
+#       additionalOptions: "--verbose --ignore-dependencies"  
```

Comment:
```suggestion
#   - task: choco
#    inputs:
#      package: "vscode"
#      version: "1.80.0"
#      ignoreChecksums: true
#      additionalOptions: "--verbose --ignore-dependencies"

#    - task: choco
#    inputs:
#      package: "azure-cli"
#      version: "2.50.0"
#      additionalOptions: "--verbose --ignore-dependencies"
```

Reply:
Two examples that I found worked well with the LLMs are:
```yaml
    examples:
      - task: choco
        description: install OracleJDK 17.0.1
        inputs:
          package: oraclejdk
          version: 17.0.1
      - task: choco
        description: install notepad++
        inputs:
          package: notepadplusplus
```